CHANGELOG.rst

```diff
@@ -2,6 +2,14 @@
 Changelog
 *********
 
+4.2.2 (2019-06-06)
```

Comment:
```suggestion
4.2.2 (unreleased)
```

I'll add the release date myself.

Reply:
Understood, I couldn't decide what I should put there, but now it seems obvious.
Thanks for your patience.